analyzer/codechecker_analyzer/pre_analysis_manager.py

```diff
@@ -203,9 +214,9 @@ def signal_handler(signum, frame):
     # Postprocessing the pre analysis results.
     if ctu_data:
         ctu_manager.merge_ctu_func_maps(
-                ctu_data.get('ctu_dir'),
-                ctu_data.get('ctu_func_map_file'),
-                ctu_data.get('ctu_temp_fnmap_folder'))
+            ctu_data.get('ctu_dir'),
+            ctu_data.get('ctu_func_map_file'),
```

Comment:
is the file name still ctu_func_map_file, shouldn't that be extdef-mapping file or something like that?

Reply:
That would make sense, however I think this change is just left here unintentionally (maybe auto-formatted accidentally :P). I will remove this part of the diff.